cutekit/project.py

```diff
@@ -1,16 +1,14 @@
 import os
 from typing import Optional
 
-
 def root() -> Optional[str]:
     cwd = os.getcwd()
-    while cwd != "/":
+    while cwd != "/" and cwd != os.path.splitdrive(cwd)[0]:
```

Comment:
I think it might be better to migrate the whole codebase to `pathlib` in order to have good compatibility with Windows

```python
cwd = Path.cwd()
while str(cwd) != cwd.root:
    ...
```

Reply:
I don't even know how to use the Python's stdlib correctly so...